packages/slice-machine/src/legacy/lib/models/common/widgets/Link/components.tsx

```diff
@@ -90,7 +90,21 @@ export function RepeatableCheckbox(props: CommonCheckboxProps) {
   );
 }
 
-export function Variant() {
+export function Variants({
+  variants,
+  onVariantsChange,
+}: {
+  variants?: string[];
+  onVariantsChange: (variants?: string[]) => void;
+}) {
+  const enabled = Boolean(variants);
+
+  const onCheckedChange = (checked: boolean) => {
+    onVariantsChange(checked ? ["Primary", "Secondary"] : undefined);
```

Comment:
💡 Potential product problem, are we sure we want to reset the variants every time you check / uncheck?

I'm wondering because, let say you're a dev with 5 variants, click accidentally or "just to see" what happens, you will lose existing data. 

I would have said that `["Primary", "Secondary"]` would only be the default if you don't have already, no?